app/models/concerns/provideable.rb

```diff
@@ -9,7 +9,7 @@ module Provideable
     validates :business_name, presence: { message: "Must provide a name or a business name" }, if: proc { |ddp| ddp.contact_name.blank? }
 
     geocoded_by :address
-    after_validation :geocode, if: ->(obj) { obj.address.present? && obj.address_changed? }
+    after_validation :geocode, if: ->(obj) { obj.address.present? && obj.address_changed? && !Rails.env.development? }
```

Comment:
We should probably pull this out into a concern

Reply:
A `geocodable` or something? I think that is probably a good idea. My inclination might be to wait until we finally start using geocoding :)

Let me make that change so whoever does work with goecodable next has one place to make changes.